projects/Scintillua++.vcxproj

```diff
@@ -20,6 +20,7 @@
   </ItemGroup>
   <ItemGroup>
     <ClInclude Include="..\src\AboutDialog.h" />
+    <ClInclude Include="..\src\LangaugeDialog.h" />
```

Comment:
The actual file is called `LanguageDialog.h` and this causes it to not be found if trying to open it through VS.